### DISCUSSION TITLE
PN532 not found

### DISCUSSION THREAD

**Discussion by electric-m** [rednblkx/HomeKey-ESP32#32] [General]
When booting, I get this error:

```
MAC Address:      FC:B4:67:93:9A:30

Device Name:      Homekey Schloss

I (2831) gpio: GPIO[5]| InputEn: 1| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (2835) gpio: GPIO[18]| InputEn: 1| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (2844) gpio: GPIO[19]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (2853) gpio: GPIO[23]| InputEn: 1| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
E (2876) NFC_SETUP: Didn't find PN53x board
Configuring LockManagement
I (2877) LockMechanism: LockMechanism > Configuring LockMechanism
I (2882) NFCAccess: NFCAccess > Configuring NFCAccess
```

I connected the PN532 exactly like in the documentation:

```
GPIO18 - SCK

GPIO19 - MISO

GPIO23 - MOSI

GPIO5 - SS
```

Where in the code are those pins defined?

**Comment by electric-m** [comment]
Also solved, power supply was not sufficient.

**Comment by pilot1981** [comment]
> When booting, I get this error:
> 
> ```
> MAC Address:      FC:B4:67:93:9A:30
> 
> Device Name:      Homekey Schloss
> 
> I (2831) gpio: GPIO[5]| InputEn: 1| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
> I (2835) gpio: GPIO[18]| InputEn: 1| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
> I (2844) gpio: GPIO[19]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
> I (2853) gpio: GPIO[23]| InputEn: 1| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
> E (2876) NFC_SETUP: Didn't find PN53x board
> Configuring LockManagement
> I (2877) LockMechanism: LockMechanism > Configuring LockMechanism
> I (2882) NFCAccess: NFCAccess > Configuring NFCAccess
> ```
> 
> I connected the PN532 exactly like in the documentation:
> 
> ```
> GPIO18 - SCK
> 
> GPIO19 - MISO
> 
> GPIO23 - MOSI
> 
> GPIO5 - SS
> ```
> 
> Where in the code are those pins defined?

how do you get this logs? I need also to troubleshoot in my lab!

thanks in advance
